templates/cmd/main.go.tpl

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/getoutreach/gobox/pkg/log"
 	"github.com/getoutreach/gobox/pkg/events"
 	"github.com/getoutreach/gobox/pkg/trace"
+	"github.com/getoutreach/stencil-golang/pkg/serviceactivities/automemlimit"
```

Comment:
uh, this doesn't exist.  you forget to `git add` some stuff?

Reply:
probably